getting-started/customize-dev-portal-example.md

```diff
@@ -0,0 +1,230 @@
+
+- At least one configured developer namespace
+- At least one operational Supply Chain
+
+To confirm the last two prerequisites, it is recommended to deploy a demo workload as described in [Deploy an app on Tanzu Application Platform](./deploy-first-app.hbs.md). 
+
+
+## <a id="you-will"></a>What you will do
+
+- Create a customized Tanzu Developer Portal containing the hello-world plugin
+- Observe the customized Tanzu Developer Portal instance
+
+## <a id="customize-dev-portal"></a>Customize your Tanzu Developer Portal by adding the hello-world plugin
+
+Complete the following steps to integrate the hello-world plugin into your Tanzu Developer Portal.
+
+### Procedure
+
+1. Create the **tpb-config.yaml** file specifying the list of additional plugins that you want to integrate. In this example, we only need to specify the hello-world plugin that has both frontend and backend parts:
+
+    ```yaml
+    app:
+      plugins:
+        - name: '@tpb/plugin-hello-world'
+          version: '^1.6.0-release-1.6.x.1'
+    backend:
+      plugins:
+        - name: '@tpb/plugin-hello-world-backend'
+          version: '^1.6.0-release-1.6.x.1'
+    ```
+> **Note:** If the plugin's version is not specified, it is likely that the workload that uses the config file as input will fail.
+
+2. Encode the file in base64, to later embed `tpb-config.yaml` in the workload definition file, by
+   running:
+
+   ```console
+   base64 -i tdp-config.yaml
+   ```
+For our example, the output shall be the following:
+```console
+YXBwOgogIHBsdWdpbnM6CiAgICAtIG5hbWU6ICdAdHBiL3BsdWdpbi1oZWxsby13b3JsZCcKICAgICAgdmVyc2lvbjogJ14xLjYuMC1yZWxlYXNlLTEuNi54LjEnIApiYWNrZW5kOgogIHBsdWdpbnM6CiAgICAtIG5hbWU6ICdAdHBiL3BsdWdpbi1oZWxsby13b3JsZC1iYWNrZW5kJwogICAgICB2ZXJzaW9uOiAnXjEuNi4wLXJlbGVhc2UtMS42LnguMScK
+```
+We will use this value later as `ENCODED-TDP-CONFIG-VALUE`.
+
+
+3. Identify the location of the configurator's image
+
+To complete this step, use the following commands:
+
+```console
+export OUTPUT_IMAGE=$(kubectl -n tap-install get package tpb.tanzu.vmware.com.0.1.2 -o "jsonpath={.spec.template.spec.fetch[0].imgpkgBundle.image}")
+```
+```console
+imgpkg pull -b ${OUTPUT_IMAGE} -o tpb-package
+```
+```console
+yq -r ".images[0].image" <tpb-package/.imgpkg/images.yml
+```
+We will use this value later as `TDP-IMAGE-LOCATION`.
+
+
+
+4. Prepare your workload definition file. Create a file called `tdp-workload.yaml` with the following content:
+
+```yaml
+apiVersion: carto.run/v1alpha1
+kind: Workload
+metadata:
+  name: tdp-configurator
+  namespace: DEVELOPER-NAMESPACE
+  labels:
+    apps.tanzu.vmware.com/workload-type: web
+    app.kubernetes.io/part-of: tdp-configurator
+spec:
+  build:
+    env:
+      - name: BP_NODE_RUN_SCRIPTS
+        value: 'set-tpb-config,portal:pack'
+      - name: TPB_CONFIG
+        value: /tmp/tpb-config.yaml
+      - name: TPB_CONFIG_STRING
+        value:
+        ENCODED-TDP-CONFIG-VALUE
```

Comment:
**issue**

those should probably be on the same line:

`value: ENCODED-TDP-CONFIG-VALUE`

Reply:
Updated